node/init.go

```diff
@@ -0,0 +1,102 @@
+package node
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/celestiaorg/celestia-node/libs/fslock"
+	"github.com/celestiaorg/celestia-node/libs/utils"
+)
+
+// Init initializes the Node FS Repository in the dir under 'path' with given Config 'cfg'.
+func Init(path string, cfg *Config) error {
```

Comment:
Question: Is it also a good practice to add more info logs during node init stage?

Reply:
Ok, will do

Reply:
Check it out